libr/include/r_cons.h

```diff
@@ -1216,8 +1216,18 @@ typedef struct r_panels_root_t {
 	RPanelsRootState root_state;
 } RPanelsRoot;
 
-#ifdef __cplusplus
+
+#ifdef __sun
+static inline void cfmakeraw(struct termios *tm) {
```

Comment:
but.. its not referenced from wanywhere :?

Reply:
cons and run uses it tought, I got linkage issue this is how I spotted it.